backends/kv/inferschema.go

```diff
@@ -33,8 +34,9 @@ import (
 )
 
 var (
-	intType   = reflect.TypeOf(1)
-	floatType = reflect.TypeOf(1.0)
+	intType            = reflect.TypeOf(1)
+	floatType          = reflect.TypeOf(1.0)
+	hashedBucketFormat = regexp.MustCompile("^[a-zA-z0-9]+_[0-9]+$")
```

Comment:
`^[a-zA-Z0-9]+_[0-9]+$`, or it will cover a bunch of other random characters between upper and low case on the ascii table.

Reply:
Also, shouldn't it be `^[a-zA-Z_][a-zA-Z0-9_]*_[0-9]+$`?

Reply:
In line with #480.